IDV changes (#6210)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/xidv/catalog-properties.md

#### OIN catalog properties

> 1. In the **OIN catalog properties** section, specify the following OIN catalog information:

Reviewer: ```suggestion
1. In the **OIN catalog properties** section, enter the following information:
```

    | <div style="width:150px">Property</div>| Description  |
    | ----------------- | ------------ |

>     | **Display name** `*` | Provide a name for your integration. This is the main title used for your integration in the OIN.<br>The maximum field length is 64 characters. |

Reviewer: ```suggestion
    | **Display name** `*` | The name of your integration, as it should appear in the OIN.<br>The maximum field length is 64 characters. |
```

>     | **Description** `*` | Give a general description of your app and the benefits of this integration to your customers. See [App description guidelines](/docs/guides/submit-app-prereq/main/#app-description-guidelines). |

Reviewer: ```suggestion
    | **Description** `*` | A description of your app and the benefits it provides to customers. See [App description guidelines](/docs/guides/submit-app-prereq/main/#app-description-guidelines). |
```

>     | **Logo** `*` | Upload a PNG, JPG, or GIF file of a logo to accompany your integration in the catalog. The logo file must be less than one MB. See [Logo guidelines](/docs/guides/submit-app-prereq/main/#logo-guidelines). |

Reviewer: ```suggestion
    | **Logo** `*` | A PNG, JPG, or GIF file of a logo to accompany your integration in the catalog. The logo file must be less than one MB. See [Logo guidelines](/docs/guides/submit-app-prereq/main/#logo-guidelines). |
```
